charts/kibana/templates/kibana-default-index-cronjob.yaml

```diff
@@ -0,0 +1,48 @@
+#######################################################
+##           Kibana Default Index CronJob        ######
+#######################################################
+apiVersion: {{ include "apiVersion.batch.cronjob" . }}
+kind: Job
+metadata:
+  name: {{ template "kibana.fullname" . }}-default-index-job
+  labels:
+    tier: logging
+    component: {{ template "kibana.name" . }}
+    chart: {{ template "kibana.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-upgrade,post-install
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  template:
+      metadata:
+        labels:
+          tier: logging
+          component: {{ template "kibana.name" . }}
+          chart: {{ template "kibana.chart" . }}
+          release: {{ .Release.Name }}
+          heritage: {{ .Release.Service }}
+          app: {{ template "kibana.name" . }}-default-index
+          version: {{ .Chart.Version }}
+      spec:
+{{- include "kibana.imagePullSecrets" . | indent 6 }}
+        containers:
+        - name: kibana-default-index
+          image: {{ template "kibana.image" . }}
```

Comment:
ap-kibana is ~325mb. Can we use ap-init (~10mb) or some other lightweight container for this? This job will most likely not run on the same node that kibana will end up running on, so pulling the kibana image is not an advantage.

Reply:
does ap-init has curl checking